src/components/Home/Home.js

```diff
@@ -18,6 +25,11 @@ class Home extends Component {
     this.props.authStateChanged()
   }
 
+  // toggle components that are shown
+  toggleComponents = el => {
```

Comment:
I think the convention to access a target must be `e` instead of `el`

```jsx
toggleComponents = e => {
	this.setState({ activeEl: e.target.id })
}
```